src/main/java/com/chartboost/sdk/sample/ChartboostSample.java

```diff
@@ -19,19 +18,8 @@
 
 public class ChartboostSample extends BaseSample {
 
-    private final String TAG = "ChartboostSample";
-
-    public static Activity activity;
-
     protected final static String impressionTypeKey = "IMPRESSION_TYPE_KEY";
 
-    // Button
-    private Button backButton;
```

Comment:
👏 👏 awesome removal, why make them class level if their scope is at the `onCreate` level 👏 👏